lib/components/RadioButton/RadioButton.styled.ts

```diff
@@ -1,57 +1,64 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
-export const RadioButtonInput = styled.input`
-  position: absolute;
-  width: ${({ theme }) => theme.spacings.md};
-  height: ${({ theme }) => theme.spacings.md};
-  margin: 0;
-  padding: 0;
+export const RadioButtonInput = styled.input<{ checked?: boolean }>`
+  display: inline-block;
+  appearance: none;
 
-  opacity: 0;
-`;
-
-export const RadioButtonRadio = styled.div<{ checked?: boolean }>`
   position: relative;
-  z-index: -1;
+  top: 3px;
   box-sizing: border-box;
   width: ${({ theme }) => theme.spacings.md};
   height: ${({ theme }) => theme.spacings.md};
+  margin: 0;
   margin-right: ${({ theme }) => theme.spacings.sm};
+  padding: 0;
 
   border: 1.5px solid
     ${({ theme, checked }) =>
       checked ? theme.colors.primary[500] : theme.colors.neutral[300]};
-  border-radius: ${({ theme }) => theme.spacings.md};
-`;
+  border-radius: 100%;
 
-export const RadioButtonMarker = styled.div`
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  box-sizing: border-box;
-  width: ${({ theme }) => theme.spacings.xsm};
-  height: ${({ theme }) => theme.spacings.xsm};
+  cursor: pointer;
+
+  &:focus {
+    outline: 1px solid transparent;
```

Comment:
don't do this. ever.
some people rely on this to be able to see it.

Reply:
I will remove this